### PR TITLE
[DOCS] Add resources dropdown to navbar

### PR DESCRIPTION
**Pull request by JessSaavedra**
# Description
- Adds resources dropdown to navbar
- Unifies help files from GX Cloud and GX Core into `how to get support`file

# Screenshots
## Navbar
Captura desde 2024-02-01 06-17-50

## On hover
image

- [ ] Description of PR changes above includes a link to [an existing GitHub issue](https://github.com/great-expectations/great_expectations/issues)
- [ ] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [ ] Code is linted - run `invoke lint` (uses `black` + `ruff`)
- [ ] Appropriate tests and docs have been updated

For more information about contributing, see [Contribute](https://docs.greatexpectations.io/docs/contributing/contributing_checklist).

After you submit your PR, keep the page open and **monitor the statuses of the various checks made by our continuous integration process at the bottom of the page. Please fix any issues that come up** and [reach out on Slack](https://greatexpectations.io/slack) if you need help. Thanks for contributing!

### CODE REVIEW

File: docs/docusaurus/docs/how_to_get_support.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+title: How to get support
```
Comment (JessSaavedra) [inline docs/docusaurus/docs/how_to_get_support.md]: This file will contain the unified info for support that will result of this documentation JIRA ticket: https://greatexpectations.atlassian.net/browse/DOC-649

I leave it here as a placeholder, but if by the time this PR is reviewed we don't have the markdown yet, I can just hide the link in the navbar and add it later!

Comment (JessSaavedra) [reply docs/docusaurus/docs/how_to_get_support.md]: File is already merged so I'm deleting this one and redirecting to the existing one!

File: docs/docusaurus/docusaurus.config.js [hunk 2]
```diff
@@ -136,6 +136,29 @@ module.exports = {
...
+        {
+          type: 'dropdown',
+          label: 'Resources',
+          items: [
+            {
+              label: 'How to get support',
+              to: '/docs/resources/get_support'
+            },
+            {
+              label: 'Expectations Gallery',
```
Comment (josectobar) [inline docs/docusaurus/docusaurus.config.js]: @kwcanuck, are we doing title case for headers?

File: docs/docusaurus/src/css/navbar.scss [hunk 3]
```diff
@@ -54,11 +55,21 @@
 
   .navbar__item {
     margin: 0 0.5em;
-    padding: 0.75em var(--ifm-navbar-padding-horizontal) 0.5em;
+    padding: 0.5em var(--ifm-navbar-padding-hoChan rizontal);
```
Comment (josectobar) [inline docs/docusaurus/src/css/navbar.scss]: typo?
```suggestion
    padding: 0.5em var(--ifm-navbar-padding-horizontal);
```

**Review by josectobar — CHANGES_REQUESTED** [review]
Looks great, just left a small change and a comment for @kwcanuck to respond.

File: docs/docusaurus/src/css/navbar.scss (hunk 3 above)
Comment (JessSaavedra) [reply docs/docusaurus/src/css/navbar.scss]: oops, totally a typo :scream: good catch, fixed!

File: docs/docusaurus/docusaurus.config.js (hunk 2 above)
Comment (kwcanuck) [reply docs/docusaurus/docusaurus.config.js]: @josectobar, we use sentence case for topic and section headings.

Comment (josectobar) [inline docs/docusaurus/docusaurus.config.js]: Lets make these labels sentence 
```suggestion
              label: 'Expectations gallery',
```

File: docs/docusaurus/docusaurus.config.js [hunk 4]
```diff
@@ -136,6 +136,29 @@ module.exports = {
...
+              to: 'https://greatexpectations.io/expectations'
+            },
+            {
+              label: 'Integration Support Policy',
```
Comment (josectobar) [inline docs/docusaurus/docusaurus.config.js]:
```suggestion
              label: 'Integration support policy',
```

**Review by josectobar — APPROVED** [review]
I left a few small casing changes but will approve so you can merge this once they've been addressed.

**Review by SuperYeehee — APPROVED** [review]

**Pull request merged**